tests/baselines/reference/user/uglify-js.log

```diff
@@ -108,6 +108,11 @@ node_modules/uglify-js/lib/scope.js(462,15): error TS2554: Expected 0 arguments,
 node_modules/uglify-js/lib/scope.js(487,15): error TS2554: Expected 0 arguments, but got 1.
 node_modules/uglify-js/lib/sourcemap.js(55,25): error TS2304: Cannot find name 'MOZ_SourceMap'.
 node_modules/uglify-js/lib/sourcemap.js(61,23): error TS2304: Cannot find name 'MOZ_SourceMap'.
+node_modules/uglify-js/lib/utils.js(124,27): error TS2339: Property 'v' does not exist on type '{}'.
```

Comment:
#27370 breaks narrowing on constructor functions, eg

```js
/** @constructor */
function AtTop(val) { this.val = val }
/** @type {*} */
var x = 1;
if (x instanceof AtTop) {
    x.val
}
```

I don't think this error is correct, but I don't understand how it arises from the change either.

Reply:
Filed #27550 to track this.

Reply:
Fix is up at #27551, so I'll close this PR.